src/components/postDetail/ImageSlide/index.tsx

```diff
@@ -0,0 +1,74 @@
+import Image from 'next/image';
+
+import { useRef, useState } from 'react';
+
+import classNames from 'classnames/bind';
+
+import { SVGS } from '@/constants';
+
+import ImageModal from '@/components/postDetail/ImageModal';
+import { IMAGE_LIST } from '@/constants/mockData/imageList';
+import useToggleButton from '@/hooks/useToggleButton';
+
+import styles from './ImageSlide.module.scss';
+
+const cx = classNames.bind(styles);
+
+const { leftArrow, rightArrow } = SVGS.button;
+const SCROLL_WIDTH = 945;
+
+type ImageSlideProps = {
+  imageList?: string[];
+};
+
+const ImageSlide = ({ imageList = IMAGE_LIST }: ImageSlideProps) => {
+  const { isVisible, handleToggleClick } = useToggleButton();
+  const [selectedImageSrc, setSelectedImageSrc] = useState('');
+  const ref = useRef<HTMLDivElement>(null);
+
+  const handleClickPrev = () => {
+    if (ref.current) {
+      ref.current.scrollLeft += SCROLL_WIDTH;
+    }
+  };
+
+  const handleClickNext = () => {
+    if (ref.current) {
+      ref.current.scrollLeft += -SCROLL_WIDTH;
+    }
+  };
+
+  const handleImageClick = (index: number) => {
+    setSelectedImageSrc(imageList[index]);
+    handleToggleClick();
+  };
+
+  const handleCloseModal = () => handleToggleClick();
+
+  return (
+    <div className={cx('image-slide')}>
+      <button className={cx('image-slide-right-btn', 'sm-hidden')} onClick={handleClickPrev}>
+        <Image src={rightArrow.url} alt={rightArrow.alt} width={20} height={20} />
+      </button>
+      <button className={cx('image-slide-left-btn', 'sm-hidden')} onClick={handleClickNext}>
+        <Image src={leftArrow.url} alt={leftArrow.alt} width={20} height={20} />
+      </button>
+      <div className={cx('container')} ref={ref}>
+        <div className={cx('container-slider')}>
+          {imageList.map((item, index) => (
+            <button
+              className={cx('container-slider-btn')}
+              key={`image-slide-${index}`}
+              onClick={() => handleImageClick(index)}
+            >
+              <img className={cx('image')} src={item} alt={`${item}`} />
+            </button>
+          ))}
+        </div>
+      </div>
+      <ImageModal imageSrc={selectedImageSrc} isOpen={isVisible} onClose={handleCloseModal} />
```

Comment:
```suggestion
      <ImageModal imageSrc={selectedImageSrc} isOpen={isVisible} onClose={handleToggleClick} />
```
46번째 줄에서 이름을 바꿔서 넣어주셨는데 prop명이 이미 onClose니까 그냥 넣어줘도 괜찮을 거 같습니다